inference-engine/thirdparty/clDNN/api/cldnn/graph/network.hpp

```diff
@@ -181,6 +184,9 @@ struct network {
                                     std::set<primitive_id> dependencies,
                                     allocation_type type,
                                     bool reusable = true);
+    std::unique_ptr<memory_pool>& get_memory_pool() {
```

Comment:
Can't you just return `memory_pool&`. I don't see where you change the unique_ptr itself (e.g. call reset for it), so there is no reason to return ref to unique_ptr instead of object ref.

Reply:
Fixed